modules/dac/model/encodec.py

```diff
@@ -0,0 +1,292 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Hi @jiaqili3, please help to revise the copyright information for such external modules